Wikipedia/Code/Panels.swift

```diff
@@ -193,6 +198,11 @@ extension UIViewController {
         }
         let panel = SyncEnabledPanelViewController(showCloseButton: true, primaryButtonTapHandler: primaryButtonTapHandler, secondaryButtonTapHandler: nil, dismissHandler: nil, theme: theme)
         let presenter = self.presentedViewController ?? self
+        if presenter is WMFThemeableNavigationController {
+            guard presenter.presentedViewController == nil else {
+                return
```

Comment:
Could we use an assert here if we expect this should never happen? (had a similar question on the related PR)

Reply:
This is the same code 😄 responded in #2255

Reply:
haha "update branch before merging"... doh!